javagen/src/main/java/com/azure/autorest/model/javamodel/JavaClass.java

```diff
@@ -199,6 +204,59 @@ public final void javadocComment(int wordWrapWidth, Consumer<JavaJavadocComment>
         contents.javadocComment(wordWrapWidth, commentAction);
     }
 
+    @Override
+    public final void javadocComment(int wordWrapWidth, Consumer<JavaJavadocComment> commentDescriptionAction, Consumer<JavaJavadocComment> commentTagsAction, boolean withGeneratedMarker) {
+        javadocComment(commentDescriptionAction, commentTagsAction, withGeneratedMarker, true, wordWrapWidth);
+    }
+
+    @Override
+    public final void javadocComment(Consumer<JavaJavadocComment> commentDescriptionAction, Consumer<JavaJavadocComment> commentTagsAction, boolean withGeneratedMarker) {
+        javadocComment(commentDescriptionAction, commentTagsAction, withGeneratedMarker, false, 0);
+    }
+
+    /**
+     * Helper method to add a javadoc comment to the contents.
+     * @param commentDescriptionAction
+     * @param commentTagsAction
+     * @param withGeneratedMarker
+     * @param withWordWrap
+     * @param wordWrapWidth
+     */
+    private void javadocComment(Consumer<JavaJavadocComment> commentDescriptionAction, Consumer<JavaJavadocComment> commentTagsAction, boolean withGeneratedMarker, boolean withWordWrap, int wordWrapWidth) {
```

Comment:
This method seems to be repeated in a few places. We should move this to a common helper method and remove duplication.